lib/api/model/narrow.dart

```diff
@@ -114,19 +116,17 @@ class ApiNarrowPmWith extends ApiNarrowDm {
   ApiNarrowPmWith._(super.operand, {super.negated});
 }
 
-// TODO: generalize into ApiNarrowIs
-class ApiNarrowIsMentioned extends ApiNarrowElement {
+class ApiNarrowIs extends ApiNarrowElement {
   @override String get operator => 'is';
-  @override String get operand => 'mentioned';
 
-  ApiNarrowIsMentioned({super.negated});
-}
+  @override final String operand;
 
-class ApiNarrowIsUnread extends ApiNarrowElement {
-  @override String get operator => 'is';
-  @override String get operand => 'unread';
+  ApiNarrowIs(IsOperand operand, {super.negated}) : operand = operand.toString();
```

Comment:
I don't really understand this paragraph:

    We convert the IsOperand value into a String when constructing
    ApiNarrowIs, so that we don't need to mingle with
    ApiNarrowElement.toJson to take care of the enum.

What would happen if we had this class store the enum?

Reply:
Testing with that again, I think I was mistaken that we need special handling for `ApiNarrowElement.toJson` to support the use of the enum field, but actually the fix is to just provide a `toJson` method on `IsOperand`.